Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -9,6 +9,54 @@ import Foundation
     case credit
 }
 
+
+/// Use this option to specify  the Upc type of the lien item.
```

Comment:
```suggestion
/// Use this option to specify  the UPC type of the line item.
```
Additionally are there more descriptive UPC code definitions somewhere that we can link to? I'm not sure what all these cases mean and assume merchants may be confused as well.

Reply:
Got it, these docs are fine then since they seem to match paypal's public docs. We can hopefully assume merchants that use these know what they mean. 🙈